src/server/script_mgr.cc

```diff
@@ -207,6 +211,16 @@ void ScriptMgr::LatencyCmd(ConnectionContext* cntx) const {
   }
 }
 
+void ScriptMgr::GCCmd(ConnectionContext* cntx) const {
+  auto cb = [](Interpreter* ir) {
+    ir->RunGC();
+    ThisFiber::Yield();
```

Comment:
why do you need `ThisFiber::Yield();` here?

Reply:
I though that if we're dropping gigabytes of memory the gc run will be expensive, so I want to avoid harsh freezes

Reply:
now I get it. your cb does preempt.